src/actions.js

```diff
@@ -2683,7 +2684,7 @@ ActionManager.prototype.onOpenProject = async function(str) {
     this.lastSeen = event.id;  // don't reset lastSeen
     this.completeAction(null, project);
 
-    if (!this.ide().isReplayMode) {
+    if (!ide.isReplayMode) {
```

Comment:
@brollb  The only question I had was if the call to `onOpenRole` should be inside or outside this if. Replay mode seems like it could break a lot of extensions (or just not make sense with them running, e.g. the robots' movement not being included in replays), but that case could be handled in the extensions themselves.

Reply:
I agree. It seems reasonable not to call it in replay mode since we aren't really opening a new role. Let's just leave it as-is and revisit the issue if a motivating example arises.